app/src/main/java/org/mozilla/webmaker/javascript/WebAppInterface.java

```diff
@@ -290,4 +292,14 @@ public void openExternalUrl(String url) {
     public boolean isDebugBuild() {
         return BuildConfig.DEBUG;
     }
+
+    /**
+     * ----------------------------------------
+     * Get System Locale
+     * ----------------------------------------
+     */
+    @JavascriptInterface
+    public String getSystemLocale() {
+        return Locale.getDefault().toString();
```

Comment:
`Locale.getDefault().getLanguage();` rather than `toString()`? Better to be intentional about what we need than the string serialization of the entire object.